# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../dvd+rwtools/sg_h_fix.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---
# Patch by Kickah
diff -Nur dvd+rw-tools-7.0.orig/transport.hxx dvd+rw-tools-7.0/transport.hxx
--- dvd+rw-tools-7.0.orig/transport.hxx	2006-12-10 23:18:18.000000000 +0000
+++ dvd+rw-tools-7.0/transport.hxx	2006-12-10 23:19:05.000000000 +0000
@@ -284,9 +284,9 @@
 #if !KERNEL_BROKEN
 		if (sg_io.masked_status&CHECK_CONDITION)
 #endif
-		{   ret = ERRCODE(sg_io.sbp);
+		{   ret = ERRCODE((unsigned char *)sg_io.sbp);
 		    if (ret==0) ret=-1;
-		    else	CREAM_ON_ERRNO(sg_io.sbp);
+		    else	CREAM_ON_ERRNO((unsigned char *)sg_io.sbp);
 		}
 	    }
 	    return ret;
